exporters/ostream/src/span_exporter.cc

```diff
@@ -10,6 +10,25 @@ namespace exporter
 {
 namespace trace
 {
+
+std::ostream &operator<<(std::ostream &os, opentelemetry::trace::SpanKind span_kind)
+{
+  switch (span_kind)
+  {
+    case opentelemetry::trace::SpanKind::kClient:
+      return os << "Client";
+    case opentelemetry::trace::SpanKind::kInternal:
+      return os << "Internal";
+    case opentelemetry::trace::SpanKind::kServer:
+      return os << "Server";
+    case opentelemetry::trace::SpanKind::kProducer:
+      return os << "Producer";
+    case opentelemetry::trace::SpanKind::kConsumer:
+      return os << "Consumer";
+  };
+  return os << "Internal";
```

Comment:
nit: perhaps `assert(false)` to ensure `SpanKind` is exhausted by above switch?

Reply:
Aborting mayn't be right option for library. I have changed it to return empty string instead. In-fact control should never reach to this line, as we are checking for all possible enum values above : )